content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,30 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Documentation content standards
+
+Kubernetes documentation does not host certain types of content:
+
+* Vendor, provider, or project-specific content from projects that are not:
+
+    * Inside the kubernetes or kubernetes-sigs GitHub organizations
+    * Inside the CNCF
+
+* Identical content hosted in multiple locations ("dual-sourced" content)
+
+For example:
+
+| Allowed |                           Action                          |                                                    Reason                                                   |
+|:-------:|:----------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+|    Y    | Linking to Kubernetes in Docker (KinD) docs from k8s docs | The KinD project resides in github.com/kubernetes-sigs.                                                     |
+|    N    | Hosting KinD docs in k8s website                          | Dual-sourced content requires duplicated effort from maintainers and tends to go stale/rotten more quickly. |
+|    Y    | Linking to Prometheus docs from k8s docs                  | Prometheus is a CNCF project.                                                                                |
+|    N    | Linking to rkt docs from k8s docs                         | rkt is an (almost) archived CNCF project.                                                                    |
+|    N    | Tutorial on using a specific product to monitor Kubernetes | Third-party product-specific content not allowed; this type of content belongs in the third-party product's documentation. |
+|    Y    | Linking to vendor-specific product docs when listing third-party products such as Ingress controllers, load-balancing products, monitoring tools, and device plugins | Avoids dual-sourced content
```

Comment:
...but only if they're inside the kubernetes or kubernetes-sigs orgs, or inside the CNCF. Consider omitting this example because it may be more confusing than helpful.

Reply:
@zacharysarah I misunderstood the extent to which vendor-specific content is not allowed until I read the Slack thread from Friday, 9 August. I'll omit the example.

Reply:
removed